### DISCUSSION TITLE
为什么启动示例执行推理的时候一直卡着，几个小时都没进度，环境是notebook t4的环境

### DISCUSSION THREAD

**Discussion by pzfok** [open-compass/opencompass#1049] [Q&A]
为什么启动示例执行推理的时候一直卡着，几个小时都没进度，环境是notebook t4的环境

**Comment by GOOD-N-LCM** [comment]
![image](https://github.com/open-compass/opencompass/assets/69232008/8b886fbe-c6ee-443c-bd86-c99360cadfec)
是类似于如此的情况吗？

**Reply by carolsos** [reply]
我也是这个问题，请问解决了吗？

**Comment by liushz** [comment]
请先确保huggingface能直接load你要测的模型
